flask_admin/contrib/sqla/view.py

```diff
@@ -1224,6 +1228,34 @@ def delete_model(self, model):
 
         return True
 
+    def import_model(self, form):
+        """
+            Import model from form.
+            :param form:
+                Form instance
+        """
+        filename = form.import_file.data.filename.lower()
+
+        if filename.endswith(".csv"):
+            imported_data = tablib.Dataset().load(form.import_file.data.stream.read().decode(), format='csv')
+        else:
+            imported_data = tablib.Dataset().load(form.import_file.data.stream.read(), format='xls')
```

Comment:
add try catch ici and use flash  message pour affich si un error de loading the csv of xls